### PR TITLE
Update WHSSDatlas_v3.jsonld

### PR DESCRIPTION
**Pull request by aeidi89**

### CODE REVIEW

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld [hunk 1]
```diff
@@ -15,8 +15,8 @@
   "copyright": null,
   "custodian": null,
   "description": null,
-  "digitalIdentifier": null,
-  "fullDocumentation": null,
+  "digitalIdentifier": "https://scicrunch.org/resolver/RRID:SCR_017124",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld]: this should be:
`"digitalIdentifier": {"@id": "https://scicrunch.org/resolver/RRID:SCR_017124"},`

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld [hunk 2]
```diff
@@ -380,20 +380,24 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Osen et al., 2019.",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld]: it is better to state full citations. you don't want people to search for the papers (at least state the doi)

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld [hunk 3]
```diff
@@ -380,20 +380,24 @@
...
   "keyword": null,
   "license": {
-    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBy4.0"
   },
   "majorVersionIdentifier": null,
   "ontologyIdentifier": null,
   "otherContribution": null,
-  "relatedPublication": null,
+  "relatedPublication": [
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld:396]: This should be:
```
"relatedPublication": [
	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.04.001"},
	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.10.017"},
	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.12.080"}
],
```

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld (hunk 3 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld:396]: Fixed

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld (hunk 2 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld]: Full citations have been added.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld (hunk 1 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld]: Fixed.

**Review by aeidi89 — COMMENTED** [review]
All changes have been implemented.

**Comment by aeidi89** [comment]
@lzehl I made the changes you requested. Can I merge this PR then?

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld [hunk 4]
```diff
@@ -380,20 +380,24 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the publications for WHS rat brain atlas v1 (Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: 10.1016/j.neuroimage.2014.04.001) and WHS rat brain atlas v3 (Osen KK, Imad J, Wennberg AE, Papp EA, Leergaard TB (2019) Waxholm Space atlas of the rat brain auditory system: Three-dimensional delineations based on structural and diffusion tensor magnetic resonance imaging. NeuroImage 199, 38-56. DOI: 10.1016/j.neuroimage.2019.05.016).",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld]: @aeidi89 thanks. I suggest to define this as rst-compatible markdown (example from the other PRs):

`"Please refer to the atlas by its RRID: SCR_017124, and cite the following publications:\n- Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: https://doi.org/10.1016/j.neuroimage.2014.04.001\n- Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2015) Addendum to "Waxholm Space atlas of the Sprague Dawley rat brain" [NeuroImage 97 (2014) 374-386]. NeuroImage 105, 561–562. DOI: https://doi.org/10.1016/j.neuroimage.2014.10.017"`

(\n means newline)

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld [hunk 5]
```diff
@@ -380,20 +380,24 @@
...
   "license": {
-    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBy4.0"
   },
   "majorVersionIdentifier": null,
   "ontologyIdentifier": null,
   "otherContribution": null,
-  "relatedPublication": null,
+  "relatedPublication": [
+	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.04.001"},
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld]: please correct the indentation (2 spaces more for each list item)

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld (hunk 4 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld]: Thanks @lzehl 
Fixed now.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld (hunk 5 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld]: Fixed.

**Review by lzehl — APPROVED** [review]

**Pull request merged**